content/doc/best-practices/tips_and_tricks.md

```diff
@@ -0,0 +1,226 @@
+---
+title: Tips & Tricks
+shortdesc: Discover some useful tips and tricks to improve your experience with Clever Cloud tools
+tags:
+- reference
+- tips
+- tricks
+keywords:
+- tips
+- tricks
+type: docs
+---
+
+Clever Cloud is an easy-to-use platform, designed with developers in mind. Whether you're a beginner or a more experienced user, we try to fit your needs bringing you the best possible experience through our web Console, Clever Tools, API, SDK, integrations.
+
+We like to pay attention to those small details, those features, that make Clever Cloud better to use, unique. Over the time, we've developed them but found out that most of our users or new team members don't know about them. This page exists to help spread the word.
+
+## Web Console
+
+Access Clever Cloud Console to manage your applications, add-ons, organizations and so on: https://console.clever-cloud.com
+
+### Menu advanced filter
+
+The left side menu is a powerful tool with a text field to filter your applications and add-ons. You can type the name of the items you want to find and see results matching your search instantly. But do you know that you can also use specific keywords? Here is the list:
+
+| Keyword        | Description |
+| -------        | ----------- |
+| `is:addon`     | Filter add-ons |
+| `is:app`       | Filter applications |
+| `is:[addon]`   | Filter add-ons (for example, `is:kv`, `is:keycloak`, `is:postgresql`) |
+| `is:[runtime]` | Filter by runtime name (for example, `is:node`, `is:php`, `is:static-apache`) |
+
+### Environment variables are multi-line
+
+When you edit an environment variable, in simple mode, you see a text input for each variable. But if you press the `Enter` key, the value splits into multiple lines. Thus, you can put the full content of a configuration file in it for example. Of course, you can do the same with expert and JSON modes, through Clever Tools or the API. Environment variables ARE multi-line.
+
+- [Clever Cloud Environment variables reference](/doc/reference/reference-environment-variables)
+
+### Validate new environment variables with ease
+
+When you create a new environment variable in simple mode, add it pressing the `Add` button or the `CTRL+Enter` keys.
```

Comment:
**suggestion:** since we get this feedback fairly often and we cannot really fix it right now
```suggestion
When you create a new environment variable in simple mode, add it pressing the `Add` button or the `CTRL+Enter` keys.

Don't forget to press the `Update changes` button at the bottom before leaving the page.
```